### DISCUSSION TITLE
I am unable to take a picture of remote schooling proof

### DISCUSSION THREAD

**Discussion by Zeroshi** [community/community#113217] [GitHub Education] [answered]
### Hi everyone,

As a student at Harvard Extension School, I have been struggling to submit my digital forms. I have tried several methods, such as printing and using my phone to capture images, but they are all inconvenient and ineffective. Thus, I would like to suggest a new feature that would allow us to upload documents directly from our PCs, such as screen captures or PDFs. I believe that this would significantly expedite the submission process and provide a much-needed convenience for students like me.

**Comment by lin20260407** [comment]
I agree with you because .pdf is more generic and better to save!

**Reply by amna12-tech** [reply]
As a student at Harvard Extension School, I've struggled with submitting digital forms. Printing and using my phone to capture images are inconvenient and ineffective. I suggest a new feature allowing document uploads directly from PCs, like screen captures or PDFs, to expedite the process and enhance convenience for students.

**Comment by jamesbiyond** [comment, marked as answer]
"Absolutely, incorporating a feature in the [Remini mod apk](https://reminiappapk.com/) to directly upload documents from PCs would be incredibly helpful. It'd streamline the submission process for students like us at Harvard Extension School. Thanks for suggesting this enhancement!"

**Reply by luq7** [reply]
Is this still under backlog? Or are we expect to see it live anytime soon?
